### ISSUE TITLE
File upload

### ISSUE THREAD

**Issue by mala2017** [RocketChat/Rocket.Chat.Java.SDK#67]
I am trying to upload a fille using uploadFIle method it gives me upload error: ErrorObject{reason='Store not found', errorType='Meteor.Error', error=0, message='Store not found [invalid-store]'}

**Comment by solodisergio** [comment]
+1

**Comment by kattukj002** [comment]
Yeah, I'm encountering the same issue right now. I'm trying to upload a file to Amazon S3, but I'm getting an error message that says "Store not found [invalid store]". Exactly what is causing this error?
